src/hotspot/cpu/riscv/c1_MacroAssembler_riscv.cpp

```diff
@@ -167,7 +167,7 @@ void C1_MacroAssembler::initialize_header(Register obj, Register klass, Register
   // This assumes that all prototype bits fitr in an int32_t
   mv(tmp1, (int32_t)(intptr_t)markWord::prototype().value());
   sd(tmp1, Address(obj, oopDesc::mark_offset_in_bytes()));
-
+// Todo UseCompactObjectHeaders
```

Comment:
Can I ask, will this pr fullly support riscv?

Reply:
@Hamlin-Li : AFAIK, porting to linux-riscv platform has not been started yet. Please let me know if anyone is interested or  has been working on it :-)

Reply:
Yes, I'm interested in it. Thanks for raising the discussion. :)

Reply:
If anybody is doing it, please send me a patch, or we can do it as a follow-up PR.

Reply:
Thanks. I'll send it to you if I finish it in time, otherwise I will do it in a separate pr.